initializr-service/src/main/resources/application.yml

```diff
@@ -91,6 +96,96 @@ initializr:
     kotlin:
       version: 1.1.3
   dependencies:
+    - name: Azure
+      content:
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          versionRange: 1.5.4.RELEASE
```

Comment:
There is a way to put the `versionRange` at group level. I'd expect the range to be the same for all versions so they should be shared in the group.